src/HHbbVV/processors/bbVVSkimmer.py

```diff
@@ -387,6 +387,8 @@ def process(self, events: ak.Array):
                         )
 
         skimmed_events["nGoodVBFJets"] = np.array(ak.sum(vbf_jet_mask, axis=1))
+        # Temp
+        skimmed_events["gen_weights"] = gen_weights
```

Comment:
Will this not be saved from the "single_weight" for loop below?

Reply:
I think this branch has a lot of bad "quick fixes" for saving the weight information which should not be kept otherwise. So would we just erase the pull request?

Yea, that part is redundant I think.